perun-core/src/main/java/cz/metacentrum/perun/core/impl/AuditMessagesManagerImpl.java

```diff
@@ -213,12 +213,12 @@ public List<AuditMessage> getMessagesByIdAndCount(PerunSession perunSession, int
 	}
 
 	@Override
-	public Paginated<AuditMessage> getMessagesPage(PerunSession perunSession, MessagesPageQuery query) {
+	public Paginated<AuditMessage> getMessagesPage(PerunSession perunSession, MessagesPageQuery query, String eventName) {
 		// take exact total count up to PAGE_COUNT_PRECISION entries, estimate it otherwise
 		return jdbc.query("select " + auditMessageMappingSelectQuery
 			+ ", case when (select count(*) from (select 1 from auditer_log limit" + PAGE_COUNT_PRECISION + ") as sample) < " + PAGE_COUNT_PRECISION + " then (select count(*) from auditer_log) else " +
 			"(select 100 * count(*) FROM auditer_log TABLESAMPLE SYSTEM (1)) end as total_count "
-			+ "from auditer_log order by id " + query.getOrder().getSqlValue() + " offset " + query.getOffset()
+			+ "from auditer_log where msg::json ->>'name' ilike '%" + eventName + "%' order by id " + query.getOrder().getSqlValue() + " offset " + query.getOffset()
```

Comment:
consider allowing a list of event names to be passed as parameter

Reply:
also, i think it would be faster to exact match the name value than to use 'like', but that requires defining available types of audit messages and passing it to GUI, where user could select them from a dropdown and in the request we would then obtain the correct type name. I think Michal mentioned on planning we could use reflection for this, or write a list of available audit messages manually.